pkg/controller/common/metrics.go

```diff
@@ -0,0 +1,64 @@
+package common
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// DefaultBindAddress is the port for the metrics listener
+	DefaultBindAddress = ":8797"
+
+	// MachineConfigControllerPausedPoolKubeletCA logs when a certificate rotation is being held up by pause
+	MachineConfigControllerPausedPoolKubeletCA = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "machine_config_controller_paused_pool_kubelet_ca",
+			Help: "a certificate rotation is waiting in specified paused pool",
```

Comment:
I do not fully understand the context of this PR, but I don't know which pool is "specified"

Reply:
Yeah, it makes more sense when the metrics actually come out of prometheus as a gauge. In the prometheus output it looks like: 
```
#HELP machine_config_controller_paused_pool_kubelet_ca a certificate rotation is waiting in specified paused pool
#TYPE machine_config_controller_paused_pool_kubelet_ca gauge
machine_config_controller_paused_pool_kubelet_ca{pool="master"} 0
machine_config_controller_paused_pool_kubelet_ca{pool="worker"} 1.670651677e+09
```

Reply:
I also don't think anyone ever actually looks at the HELP but I can't be sure, so I fill it in.